src/utils/userinfo.rs

```diff
@@ -85,10 +85,20 @@ pub async fn userinfo_fn<'a>(
         emoji_data.update_emoji_cache_poise(&ctx, false).await?;
     }
 
-    let (nickname, joined_at, roles) = match ctx.guild() {
+    let (nickname, joined_at, roles, status) = match ctx.guild() {
         Some(guild) => {
+            let mut status = "Not a member".to_string();
```

Comment:
I'd rather have this in the form of

```rs
let status = if ... { ... } else { ... };
```

and moved to the code path where it is actually relevant (The `Err(_)` arm where it is used).